package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "codepirates-ah-frontend",
+  "version": "1.0.0",
+  "description": "A blogging platform",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack --env.environment=production --config webpack.config.js && node server/app.js",
+    "build": "webpack --env.environment=production --config webpack.config.js",
```

Comment:
Before building. It will be a good idea to remove the dist folder from here

Reply:
- Added shell command to delete any existing dist folder before new build
- Moved webpack to project dependancies
- Refactored start and build scripts